apps/web/src/common/constants.ts

```diff
@@ -37,5 +37,13 @@ export const sidebarNavItems: SidebarButtonProps[] = [
   },
 ];
 
-export const PlanningSteps = ["Profile", "Care Activities Bundles", "Occupation", "Activities Gap", "Suggestions"];
+export const PlanningSteps = [
+  'Profile',
+  'Care Activities Bundles',
+  'Occupation',
+  'Activities Gap',
+  'Suggestions',
+];
 
+export const formFormatting =
```

Comment:
Could you move this to styles.ts and name more appropriately. formFormatting is too generic

Reply:
Done